lib/src/facts/linux/operating_system_resolver.cc

```diff
@@ -143,6 +143,9 @@ namespace facter { namespace facts { namespace linux {
         }
 
         auto release = implementation->get_release(result.name, result.distro.release);
+        if(result.name == os::debian && !release.empty()){
```

Comment:
You can move this under the `if` below, since we already check for `!release.empty()`.

```cpp
if (!release.empty()) {
	if (result.name == os::debian) {  // on debian 10, lsb_release version is different from /etc/debian_version
		result.distro.release = release;
    }
    result.release = move(release);
    tie(result.major, result.minor) = implementation->parse_release(result.name, result.release);
}
```